code/modules/shuttle/shuttle.dm

```diff
@@ -844,25 +848,23 @@
 /obj/machinery/computer/shuttle/ferry/request
 	name = "ferry console"
 	circuit = /obj/item/circuitboard/ferry/request
-	var/cooldown //prevents spamming admins
+	var/next_request	//to prevent spamming admins
 	possible_destinations = "ferry_home"
-	admin_controlled = 1
+	admin_controlled = TRUE
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 
-/obj/machinery/computer/shuttle/ferry/request/Topic(href, href_list)
+/obj/machinery/computer/shuttle/ferry/request/tgui_act(action, params)
```

Comment:
Did you mean `can_call_shuttle()` here? Youre making the parent TGUI call here, which is going to cause issues.

Reply:
Can you explain _why_ it's going to cause issues? I don't see why it should be can_call_shuttle

Reply:
After doing some thourough testing, this *shouldnt* circumvent security measures, but is still a little confusing given that 
```dm
if(..())
    return
```
is the standard for security checking, which while this does do, it does a lot more than that. May want to add a comment clarifying this.